metricsaggregator/src/main/java/io/dockstore/metricsaggregator/client/cli/MetricsAggregatorClient.java

```diff
@@ -117,20 +123,38 @@ public static void main(String[] args) {
                 try {
                     final MetricsAggregatorConfig metricsAggregatorConfig = new MetricsAggregatorConfig(config.get());
                     metricsAggregatorClient.submitValidationData(metricsAggregatorConfig, submitValidationData.getValidator(),
-                            submitValidationData.getValidatorVersion(), submitValidationData.getDataFilePath(), submitValidationData.getPlatform());
+                            submitValidationData.getValidatorVersion(), submitValidationData.getDataFilePath(),
+                            submitValidationData.getPlatform());
                 } catch (Exception e) {
                     LOG.error("Could not submit validation metrics to Dockstore", e);
                     System.exit(FAILURE_EXIT_CODE);
                 }
             }
+        } else if ("submit-terra-metrics".equals(jCommander.getParsedCommand())) {
+            if (submitTerraMetrics.isHelp()) {
+                jCommander.usage();
+            } else {
+                final Optional<INIConfiguration> config = getConfiguration(submitTerraMetrics.getConfig());
+                if (config.isEmpty()) {
+                    System.exit(FAILURE_EXIT_CODE);
```

Comment:
Was like this before, but try to 
a) group the "System.exit" calls so it's easy for a debugger (future you or me) to place a breakpoint on it. Tracking down a random System.exit is a pain since the whole VM shuts down (aborting tests in surefire for example)
b) add a `LOG.error` like below to give some clues where to find this exit call

Reply:
Examples from CLI repo
https://github.com/dockstore/dockstore-cli/blob/b1c1300e3c6af0fcc1289b817795cf26c6294265/dockstore-client/src/main/java/io/dockstore/client/cli/ArgumentUtility.java#L106
https://github.com/dockstore/dockstore-cli/blob/b1c1300e3c6af0fcc1289b817795cf26c6294265/dockstore-client/src/main/java/io/dockstore/client/cli/Client.java#L799

Reply:
Made changes so that `System.exit` is only called from methods in `ExceptionHandler`, an existing helper class used by the tooltester.